LICENSE

```diff
@@ -3,7 +3,7 @@ MIT License
 Copyright (c) 2012-2018 MIPT-ILab, Alexander Titov, Pavel Kryukov,
 Igor Smirnov, Igor Burovenko, Kamil Garifullin, Oleg Ladin, Victor Prutyanov,
 Ivan Korostelev, Alex Seppar, George Korepanov, Alexey Steksov, Kirill Nedostoev,
-Alexander Misevich, Denis Los, Alexander Timofeev, and Yan Logovskiy.
+Alexander Misevich, Denis Los, Alexander Timofeev, Soshin Konstantin and Yan Logovskiy.
```

Comment:
Please use Oxford comma and Western name order.

Reply:
(Use Western name order everywhere. Non-Russian speakers would treat 'Soshin' as given name and 'Konstantin' as family name)